tests/console/verify_firewall_services.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# Summary: Verification of firewall being inactive and allowing services http https.
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use parent 'y2_module_consoletest';
+use testapi;
+
+sub run {
+    validate_script_output("systemctl is-active firewalld",                        sub { /inactive/ });
```

Comment:
I was too fast to merge, your VR has failed cause of this line, as this command returns non-zero code. If documentation is right, it should be enough to add `proceed_on_failure` flag: http://open.qa/api/testapi/#_script_output as `validate_script_output` is a wrapper on this method.

Reply:
Yes, sorry I forgot to use [WIP] until VR was done.
I pushed : https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11854

Reply:
No worries ;). Merged, thanks for the quick fix!